wow_auto_undercut_update.py

```
@@ -6,29 +6,36 @@
     # Specify the base directory and target Lua file name
     #   ex: r"E:\World of Warcraft\_retail_\WTF\Account\12345678#2"
-    config_path = os.path.join(os.getcwd(), "wow_user_data", "undercut", "addon_undercut.json")
-    with open(config_path, 'r', encoding='utf-8') as file:
+    config_path = os.path.join(
+        os.getcwd(), "wow_user_data", "undercut", "addon_undercut.json"
+    )
+    with open(config_path, "r", encoding="utf-8") as file:
         config = json.load(file)
-        base_directory = config.get("base_directory")  # Using .get() to avoid KeyError if the key doesn't exist
+        base_directory = config.get(
+            "base_directory"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)